src/selections/expr.cpp

```diff
@@ -430,8 +430,8 @@ optional<double> Number::optimize() {
 }
 
 std::string Number::print() const {
-    if (lround(value_) == value_) {
-        return std::to_string(lround(value_));
+    if (static_cast<double>(std::lround(value_)) == value_) {
```

Comment:
Would `std::round` be better than `std::lround` with a cast?

Reply:
Yes, good catch